src/Microsoft.Health.Core/Microsoft.Health.Core.csproj

```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Messaging.EventGrid" Version="4.3.0" />
```

Comment:
I don't think this belongs here either. To follow the common patterns we have set, I would expect there to be a `Microsoft.Health.EventGrid` project that would hold `EventGrid` implementations of the various parts you're trying to implement.

Reply:
I see. Okay. Let me add it there then.